test/functional/tools/explicit_conversion.xml

```diff
@@ -0,0 +1,23 @@
+<tool id="explicit_conversion" name="explicit_conversion">
+  <command>
+    cut -f 1 '${input1_table}' > 'col_output'
```

Comment:
Is there a preference toward not using ``from_work_dir``? I don't really know which I find better - and  I think this tool was just copied from ``implicit_conversion.xml`` which was copied from ``column_param.xml``.